.github/workflows/build-docker-images-race.yml

```diff
@@ -0,0 +1,594 @@
+# Copyright 2023 UMH Systems GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build Docker Images (Race checker)
+
+on:
+  push:
+    paths:
+      - 'deployment/**'
+      - '!deployment/*.ps1'
+      - 'golang/**'
+      - '.github/workflows/build-docker-images.yml'
```

Comment:
I don't think this is intended. Should it be `- '.github/workflows/build-docker-images-race.yml'?